ImageNet/train/fastai_pytorch.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "v1.0",
+    "author": "fast.ai + students team: Jeremy Howard, Andrew Shaw, Brett Koonce, Sylvain Gugger",
+    "authorEmail": "j@fast.ai",
+    "framework": "fastai / pytorch",
+    "codeURL": "https://github.com/fastai/imagenet-fast/blob/1bc5aeec765572397c0ebe4a5d616d03beeeeec1/aws/do_nosh.sh",
+    "model": "Resnet 50",
+    "hardware": "8 * V100 (AWS p3.16xlarge)",
+    "costPerHour": 24.48,
+    "timestamp": "2018-04-20",
+    "misc": {
```

Comment:
What hyperparameters did you use?  Please add the following information to `misc`:

- learning rate schedule
- batch size
- optimizer
- momentum
- weight decay

The [Google ResNet50 submission](https://github.com/stanford-futuredata/dawn-bench-entries/blob/master/ImageNet/train/google_resnet50_tpu_tensorflow18.json) is a good example for reference.